identity_credential/src/credential/jws.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// A wrapper around a JSON Web Signature (JWS).
+pub struct Jws(String);
+
+impl Jws {
+  /// Creates a new `Jws`.
+  pub fn new(jwt_string: String) -> Self {
+    Self(jwt_string)
+  }
+
+  /// Returns a reference of the JWS string.
+  pub fn as_string(&self) -> &String {
```

Comment:
```suggestion
  pub fn as_str(&self) -> &str {
```

Conventionally we use `as_str` and return `&str`.